components/checkbox/README.md

```diff
@@ -8,66 +8,67 @@ author: orinevares
 ![Status](https://img.shields.io/badge/Component-Draft-orange.svg)
 
 # Checkbox
-Checkboxes are a form of control inputs that allow users to select one or more options from a list.
+Checkboxes are a form of control inputs that allow users to select one or more items from a list.
 
 ## Required
 
-## Last Updated: 2 days ago
+## Last Updated:
```

Comment:
are we not providing updates?

Reply:
Yes, just removing the placeholder text for now

Reply:
gotchya